web/themes/custom/pineapple/components/_button.twig

```diff
@@ -1,6 +1,6 @@
 {% set button_attributes = button_attributes|default(create_attribute()).addClass([
-  'button',
-  button_type|default('button--primary'),
+  'btn',
+  button_type|default('btn--primary'),
```

Comment:
I think you should update this to ```'btn--` ~ button_type|default('primary')``` then in that case button type would simply be the word either primary, secondary ect

Reply:
done